content/en/docs/reference/config/analysis/ist0113/index.md

```diff
@@ -0,0 +1,128 @@
+service named `my-service` in namespace `my-namespace`. To determine which
+policy object is applied to `my-service`, the following resources are matched in
+order:
+
+1. A policy resource in namespace `my-namespace` that contains a `target`
+   specifying `my-service`.
+1. A policy resource in namespace `my-namespace` that does not contain a
+   `target`. This policy implicitly applies to the entire namespace.
+1. A mesh policy resource named `default`.
+
+To determine which destination rules are applied to traffic sent to
+`my-service`, we must first know which namespace the traffic originates from.
+For the sake of this example, let's call that namespace `other-namespace`.
+Destination rules are matched in the following order:
+
+1. A destination rule in namespace `other-namespace` that specifies a host that
+   matches `my-service.my-namespace.svc.cluster.local`. This may be an exact
+   match or a wildcard match. Also note that the `exportTo` field, which
+   controls the visibility of configuration resources, will effectively be
+   ignored as resources within the same namespace as the source service are
+   always visible.
+1. A destination rule in namespace `my-namespace` that specifies a host that
+   matches `my-service.my-namespace.svc.cluster.local`. This may be an exact
+   match or a wildcard match. Note that the `exportTo` field must specify that
+   this resource is public (e.g. it has the value `"*"` or is not specified) in
+   order for a match to occur.
+1. A destination rule in the "root namespace" (which is, by default,
+   `istio-system`) that matches `my-service.my-namespace.svc.cluster.local`. The
+   root namespace is controlled by the `rootNamespace` property in the
+   [`MeshConfig` resource](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig).
+   Note that the `exportTo` field must specify that this resource is public
+   (e.g. it has the value `"*"` or is not specified) in order for a match to
+   occur.
+
+Finally, be aware that Istio doesn't apply any concept of inheritance when
+following these rules. The first resources that matches the specified criteria
+is used.
+
+## How to resolve
+
+Look at the output of the message. You should see something like:
+
+{{< text plain >}}
+Error [IST0113] (DestinationRule default-rule.istio-system) A DestinationRule
+and Policy are in conflict with regards to mTLS for host
+myhost.my-namespace.svc.cluster.local:8080. The DestinationRule
+"istio-system/default-rule" specifies that mTLS must be true but the Policy
+object "my-namespace/my-policy" specifies Plaintext.
+{{< /text >}}
+
+Contained in this message are the two resources that are in conflict:
+
+* Policy resource `my-namespace/my-policy`, which is specifying 'Plaintext' as its
+  supported mutual TLS mode.
+* Destination rule resource `istio-system/default-rule`, which is requiring mutual TLS
+  when sending traffic to host `myhost.my-namespace.svc.cluster.local:8080`
+
+You can fix the conflict by:
```

Comment:
Maybe rephrase to make it clear that the user needs to do one of the following, not all of the following.

e.g. "You can fix the conflict in several ways:"

 It's a bit unclear which 
 add "doing one of the following" or "doing all of the following" to the end of this?

Reply:
Added "doing one of the following". Thanks!